src/components/components.css

```diff
@@ -0,0 +1,19 @@
+.fixed-bottom-right {
```

Comment:
Don't create this `.css` file, we are using `Styled-Components` for styling, stick with it.

Reply:
okay